aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.csv

```diff
@@ -14,7 +14,9 @@ aircraft:design:drag_increment,0.00175,unitless
 aircraft:design:equipment_mass_coefficients,928,0.0736,0.112,0.14,1959,1.65,551,11192,5,3,50,7.6,12,unitless
 aircraft:design:max_structural_speed,402.5,mi/h
 aircraft:design:part25_structural_category,3,unitless
-aircraft:design:reserves,4998,unitless
+aircraft:design:reserves,4998,lbm
+aircraft:design:reserves_fraction,0,unitless
+aircraft:design:reserves_option,1,unitless
 aircraft:design:static_margin,0.03,unitless
```

Comment:
Thinking about the API here. There are 3 variables to specify now - "reserves", "reserves_fraction", and "reserves_option". Depending on how they set the option, one will be used and the other won't be used. I think this might cause some confusion.

What if we just had 2 variables to set?

reserves
reserves_fraction

The user can set one or the other. If they set both, then we raise an error in the pre-processor. I think this would be concise and unambiguous. We can process the logic based on which one they set.

Reply:
I thought about this approach but decided not to go this way. There are two reasons:

1. GASP actually have three cases. In addition to the above two, there is another fraction for time. It maps (0, 1) to (0, 45 minutes). Jeff Bowles told me that it was important and shouldn't be missed. I think it is for reserve missions and we have not implemented. Your approach will be complicate when we add the third case.
2. I was thinking that users may want to set both  'Aircraft.Design.RESERVES' and  'Aircraft.Design.RESERVES_FRACTION' to some reasonable values and use  'Aircraft.Design.RESERVES_OPTION' to switch back and force.

Reply:
That makes sense. I guess when that case is added, we would have another value for reserves_option.

Reply:
The third case definitely should not be set in Aviary using a (0,1) -> (0,45 min) variable, and should instead be handled the way we currently set up the design mission.

I disagree that Ken's suggestion makes things harder for the future, I have more thoughts in my overall review comments.

Reply:
Now, I excluded the potential implementation of the third case here.